dom/dom_c.php

```diff
@@ -162,14 +162,15 @@ class DOMNode
     /**
      * Adds a new child before a reference node
      * @link https://php.net/manual/en/domnode.insertbefore.php
-     * @param DOMNode $node <p>
+     * @template TNode of DOMNode
+     * @param TNode $node <p>
      * The new node.
      * </p>
      * @param null|DOMNode $child [optional] <p>
      * The reference node. If not supplied, newnode is
      * appended to the children.
      * </p>
-     * @return DOMNode The inserted node.
+     * @return TNode|false The inserted node.
```

Comment:
According to the phpdoc and https://www.php.net/manual/en/domnode.insertbefore.php
it returns the node inserted, which is the one coming from the param.

AND `false` was missing